ground/src/main/java/com/google/android/ground/persistence/local/room/dao/UserDao.kt

```diff
@@ -25,4 +25,6 @@ interface UserDao : BaseDao<UserEntity> {
   @Transaction
   @Query("SELECT * FROM user WHERE id = :id")
   suspend fun findById(id: String): UserEntity?
+
+  @Query("SELECT * FROM user ORDER BY id ASC LIMIT 1") suspend fun getUserDetails(): UserEntity
```

Comment:
This works because we currently only allow one user in the db at a time, but `UserDao` doesn't know that. In any case, you won't need to call `findById` either; see my comment that follows.